ports/q2pro/port.json

```
@@ -0,0 +1,31 @@
+{
+  "version": 3,
+  "name": "Q2Pro.zip",
```

```suggestion
  "name": "q2pro.zip",
```
The zip filename should stay lowercase to match the port's foldername